qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetDataModelProjectTest.java

```diff
@@ -436,7 +436,7 @@ public void checkedTemplateWithDefaultName() throws Exception {
 		List<DataModelParameter> helloParameters = helloTemplate.getParameters();
 		Assert.assertNotNull(helloParameters);
 
-		Assert.assertEquals(2, helloParameters.size());
+		Assert.assertEquals(1, helloParameters.size());

```

Comment:
The PR fixes a bug with the test which uses:

```java
@CheckedTemplate(defaultName=CheckedTemplate.HYPHENATED_ELEMENT_NAME)
	static class Templates2 {
		static native TemplateInstance HelloWorld(String name);

	}

```

JDT Java search engine reports 2 search matches:

 * one for CheckedTemplate
 * one for CheckedTemplate.HYPHENATED_ELEMENT_NAME

which explains the duplicate.

/cc @rgrunber perhaps it is a bug of JDT?

My fix  is just checking that parameter exists before add it.